OP2Maps/MapHeader.cpp

```diff
@@ -0,0 +1,8 @@
+#include "MapHeader.h"
+
+#include <cmath>
+
+unsigned int MapHeader::MapTileWidth()
+{
+	return (unsigned int)pow(2, lgMapTileWidth);
```

Comment:
You can avoid cmath using the formula:
tileWidth = 1 << lgTileWidth

The above formula compiles down to a single machine instruction. It's valid for twos complement integer operations, which is basically everything outside of floating point. It starts with 1 (a single low order bit set), and shifts it left by the specified number, effectively multiplying it by 2 to the power of that number. This is analogous to multiplying by powers of 10 by adding zeros to the right. A trick most programmers will be familiar with.

Using pow results in a call to a complicated looping function, resulting in more compiled code, and less efficiency. It's use is mainly for non-integer exponents. C++ does have overrides (which C does not support) for greater efficiency when the second parameter is an integer (a convenience for large powers), though it's still a general algorithm, and the compiler won't be able to optimize for the specifics of having a fixed base of 2. It's often advised to avoid using pow unless you need the full generality it offers.